### PR TITLE
Added two quizzes about vim

### PR DESCRIPTION
**Pull request by cfculler**
Added a simple quiz that tests basic knowledge about different Vim modes, and another that dives deeper into the Normal mode shortcuts

### CODE REVIEW

File: src/quizzes/chapter-1/1.3.2/vim-specifics-quiz.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+	> This will jump your cursor to the beginning of the line no matter where it's at
+- [ ] o
+	> This will insert a new line below the current line no matter where the cursor is at
+- [ ] O
+	> This will insert a new line above the current line no matter where the cursor is at
+- [ ] A
+	> This will jump your cursor to the end of the line (and enter insert mode) no matter where it's at
+- [ ] yy
+	> This will copy the current line no matter where your cursor is at
+`;
```
Comment (jburns24) [inline src/quizzes/chapter-1/1.3.2/vim-specifics-quiz.js:58]: Good questions! Might I suggest a couple more that I find really helpful when navigating a file


Certainly! Vim is a powerful text editor, and knowing its basic commands, especially for movement, searching, and find/replace, is crucial. Below are a few questions focused on these aspects of Vim, each designed to test understanding of basic yet essential Vim commands.


## Which of the following commands in Vim moves the cursor to the beginning of the file?

 [x] gg
 [] G - Move cursor to the first character of the last line of the file
 [] ^ - Moves to the first non-blank character of the current line
 [] $ - Moves to the end of the current line



## In Vim, how do you search for the word 'error' in the text?

[] :error
[x] /error
[] ?error
[] *error

File: docs/1-introduction/1.3.2-vim.md [hunk 2]
```diff
@@ -109,6 +109,16 @@ Normal mode.
 
 ?> To learn more about Vim, use the command `vimtutor` in your terminal to run through a tutorial that explores more advanced commands.
 
+Time for a few simple quizzes. Let's start off by making sure you're familiar with the basic mode differences
+<div class="quizdown">
+  <div id="chapter-1/1.3.2/vim-modes-quiz.js" ></div>
+</div>
+
```
Comment (jburns24) [inline docs/1-introduction/1.3.2-vim.md:116]: Feels like these quizzes should be separated as two back to back quizzes looks kind of bad. Maybe we just make one larger quiz. Initially I was thinking to move this one up right under the section on Modes but that would make it too easy I think.

Comment (cfculler) [reply docs/1-introduction/1.3.2-vim.md:116]: I originally wanted the first 4 questions about the modes to take place before the harder questions, but still wanted to randomize the order of the questions. I looked into it, and you can't randomize the question order while still having the same 4 questions take place before the rest. I'll just combine the quizzes and not shuffle the question order

File: src/quizzes/chapter-1/1.3.2/vim-specifics-quiz.js (hunk 1 above)
Comment (cfculler) [reply src/quizzes/chapter-1/1.3.2/vim-specifics-quiz.js:58]: I like it, I'll add these questions and add the new commands earlier in the section

Comment (cfculler) [reply src/quizzes/chapter-1/1.3.2/vim-specifics-quiz.js]: Added 👍

File: docs/1-introduction/1.3.2-vim.md (hunk 2 above)
Comment (cfculler) [reply docs/1-introduction/1.3.2-vim.md]: Combined 👍

**Review by jburns24 — APPROVED** [review]

**Pull request merged**